src/pages-conditional/dapps.js

```diff
@@ -993,7 +993,7 @@ const DappsPage = ({ data }) => {
       ),
       url: "https://uniswap.exchange/swap",
       image: data.uniswapec.childImageSharp.fixed,
-      alt: "Uniswap Logo",
+      alt: `Uniswap ${translateMessageId("logo", intl)}`,
```

Comment:
We should reference a `"uniswap-logo"` and include the full string `"Uniswap logo"` as in this commit for stablecoins: https://github.com/ethereum/ethereum-org-website/pull/1938/commits/842088053f17f65d199ba884741d3cdc2c02811b

I kept the strings to just `product-logo` so we can reuse them across pages.

Reply:
But in that case wouldn't we need different translation strings for every product? I don't think it's worth our translator's time to translate a bunch of those ("Uniswap logo", "Darkforest logo", "Pooltogether logo", etc.). In general I don't think proper nouns like company names should be put into translation files.

Reply:
But this string isn't just the product name, it includes "logo"... this implements a structure and makes it a sentence which will differ in structure based on language. If we want to do localisation as properly as possible, I believe we should include these as strings. It's not a lot of extra work IMHO.